src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -373,56 +373,13 @@ private async ValueTask<EvaluationEvent> EvaluateFeature<TContext>(string featur
                     Activity.Current != null &&
                     Activity.Current.IsAllDataRequested)
                 {
-                    AddEvaluationActivityEvent(evaluationEvent);
+                    TelemetryEventHandler.HandleEvaluationEvent(evaluationEvent, Logger);
```

Comment:
```suggestion
     FeatureEvaluationTelemetry.Publish(evaluationEvent, Logger);
```
How does this one sound?

Reply:
I like it. Updated